app/src/main/java/nl/hogeschoolrotterdam/projectb/LocationEditActivity.java

```diff
@@ -0,0 +1,135 @@
+import com.google.android.gms.maps.*;
+import com.google.android.gms.maps.model.CameraPosition;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by maartendegoede on 20/03/2019.
+ * Copyright © 2019 Anass El Mahdaoui, Hicham El Marzgioui, Michaël van Asperen, Wesley de Man, Maarten de Goede all rights reserved.
+ */
+public class LocationEditActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+    GoogleMap mGoogleMap;
+    MapView mMapView;
+    private EditText mSearchText;
+    private static final String TAG = "Location";
+    private Button button;
+    LatLng latLng;
+
+
+    private void inity() {
+        mGoogleMap.setOnMapLongClickListener(new GoogleMap.OnMapLongClickListener() {
+
+            @Override
+            public void onMapLongClick(LatLng Longl) {
+                latLng = Longl;
+                setMarker(latLng);
+
+            }
+        });
+        mSearchText = findViewById(R.id.input_search);
+        mSearchText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView textView, int actionId, KeyEvent event) {
+                if (actionId == EditorInfo.IME_ACTION_SEARCH
+                        || actionId == EditorInfo.IME_ACTION_DONE
+                        || event.getAction() == KeyEvent.ACTION_DOWN
+                        || event.getAction() == KeyEvent.KEYCODE_ENTER) {
+                    geoLocate();
+                }
+                return false;
+            }
+        });
+    }
+
+    private void geoLocate() {
+        String searchingString = mSearchText.getText().toString();
+        Geocoder geocoder = new Geocoder(this);
+        List<Address> list = new ArrayList<>();
+        try {
+            list = geocoder.getFromLocationName(searchingString, 1);
+
+
+        } catch (IOException e) {
+        }
+        if (list.size() > 0) {
+            Address address = list.get(0);
+            Log.d(TAG, "Locate adress:" + address.toString());
+            //Toast.makeText(this, address.toString(), Toast.LENGTH_SHORT).show();
+            CameraPosition search = CameraPosition.builder().target(new LatLng(address.getLatitude(),
+                    address.getLongitude())).zoom(10).bearing(0).tilt(0).build();
+            mGoogleMap.moveCamera(CameraUpdateFactory.newCameraPosition(search));
+            latLng = new LatLng(address.getLatitude(), address.getLongitude());
+            setMarker(latLng);
+
+
+        }
+    }
+
+    private void setMarker(LatLng latLng) {
+        mGoogleMap.clear();
+        MarkerOptions options = new MarkerOptions().position(latLng);
+        mGoogleMap.addMarker(options);
+
+    }
+
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
```

Comment:
Above `super.onCreate(savedInstanceState);`, please add `setTheme(WhibApp.getInstance().getThemeId());` to correctly set the theme of the activity.